package.json

```diff
@@ -39,10 +39,17 @@
     ]
   },
   "dependencies": {
+    "@google/cloud-debug": "^0.8.3",
+    "@google/cloud-trace": "^0.5.5",
+    "@google-cloud/datastore": "^0.1.1",
+    "@google-cloud/storage": "^0.1.1",
+    "@google-cloud/pubsub": "^0.2.0",
```

Comment:
@jmdobry Is this still an issue after our discussion about unifying dependencies yesterday?